scripts/docstring.sh

```diff
@@ -7,5 +7,3 @@ DOCSTRING_SRC=${1:-"src/zenml tests/harness"}
 export ZENML_DEBUG=1
 export ZENML_ANALYTICS_OPT_IN=false
 interrogate $DOCSTRING_SRC -c pyproject.toml --color
-pydocstyle $DOCSTRING_SRC -e --count --convention=google --add-ignore=D403
```

Comment:
How about we get rid of interrogate as well? I think the other docstrings checks (pydocstyle, darglint) already covered similar/better metrics which should now be handled by ruff?

Reply:
Good point. Yes I think we can get rid of it completely as well.